src/controllers/writingsController.js

```diff
@@ -0,0 +1,12 @@
+const { writingsService } = require('../services');
+const { catchAsync } = require('../utils/err');
+
+const searchTitle = catchAsync(async (req, res) => {
+    const { searchWord } = req.query;
+    if (searchWord) {
+        const result = await writingsService.searchTitle(searchWord);
+        return res.status(200).json({ result });
+    } else return res.status(204).json({});
```

Comment:
```suggestion
    if (!searchWord) {
        return res.status(204).end();
        // 혹은 기획에 따라 에러를 반환할 수도
    }   
     
    const result = await writingsService.searchTitle(searchWord);
    return res.status(200).json({ result });    
```
if else로 작성하여 서로에게 묶여있는 구조로 코드를 작성하는 것보다 다음과 같이 수정하면 더 좋을 것 같습니다.
이런식으로 예외처리가 필요한 경우 미리 해당 경우를 리턴해주는 패턴을 early return 패턴이라고 합니다.